openwhisk/openwhisk_apigateway.md

```diff
@@ -131,62 +197,44 @@ curl -X GET https://2ef15285-gws.api-gw.mybluemix.net/club/books
 ok: created api /books post for action deleteBook
-https://2ef15285-gws.api-gw.mybluemix.net/club/books
+https://service.us.apiconnect.ibmcloud.com/gws/apigateway/api/21ef035/club/books
 ok: created api /books put for action deleteBook
-https://2ef15285-gws.api-gw.mybluemix.net/club/books
+https://service.us.apiconnect.ibmcloud.com/gws/apigateway/api/21ef035/club/books
 ```
-{: screen}
 
 We can verify that the API has been re-created
 ```
-wsk api-experimental list /club
+wsk api list /club
 ```
-{: pre}
 ```
 ok: apis
 Action                    Verb         API Name        URL
-getBooks                   get         Book Club       https://2ef15285-gws.api-gw.mybluemix.net/club/books
-postBooks                 post         Book Club       https://2ef15285-gws.api-gw.mybluemix.net/club/books
-putBooks                   put         Book Club       https://2ef15285-gws.api-gw.mybluemix.net/club/books
-deleteBooks             delete         Book Club       https://2ef15285-gws.api-gw.mybluemix.net/club/books
-```
-{: screen}
-
-- **Note**: This feature is currently an experimental offering that enables users an early opportunity to try it out and provide feedback. The following feedback has already been received:
-  - No ability to customize HTTP access control for Cross-Origin Resource Sharing (CORS); currently, the generated API response headers are configured to allow any HTTP verb or origin (i.e. *). The following headers are always retturned:
-    - Access-Control-Allow-Origin: *
-    - Access-Control-Allow-Headers: Authorization, Content-Type
-    - Access-Control-Allow-Methods: GET, POST, PUT, DELETE, PATCH, HEAD, OPTIONS
-  - Only Content Type `application/json` is supported for request and response.
-  - No programatic way to control the response from the OpenWhisk action.
-  - All OpenWhisk actions are expose via public access, no ability to configure a custom API key.
-  - Path parameters are not supported, only query parameter and request body.
-  - If the API is created without an API name, the name will be the base path and this cannot be changed
-  - When re-creating APIs via input file, the API needs to be deleted first.
-  - When exporting APIs this contain your OpenWhisk API key, this information is sensitive, no templating available.
+getBooks                   get         Book Club       https://service.us.apiconnect.ibmcloud.com/gws/apigateway/api/21ef035/club/books
+postBooks                 post         Book Club       https://service.us.apiconnect.ibmcloud.com/gws/apigateway/api/21ef035/club/books
+putBooks                   put         Book Club       https://service.us.apiconnect.ibmcloud.com/gws/apigateway/api/21ef035/club/books
+deleteBooks             delete         Book Club       https://service.us.apiconnect.ibmcloud.com/gws/apigateway/api/21ef035/club/books
+```
+
+- **Note**: The `wsk api-experimental` CLI command will be available for a short period of time to allow you to migrate and delete your existing APIs.
```

Comment:
Do we need this again?  It's already in the leading section and seems a bit out of place here.